### PR TITLE
feat: use react-attach-handler module instead of own event attaching …

### PR DESCRIPTION
**Pull request by lucidNTR**
…and use id scopes for the sliders key events

### CODE REVIEW

File: src/GallerySwiper.jsx [hunk 1]
```diff
@@ -1009,9 +936,24 @@ class GallerySwiper extends Component {
             }
         });
 
+        let events;
+        if (!disableArrowKeys) {
+            events = (
+                <AttachHandler
+                    target={BASE_CLASS}
```
Comment (lucidNTR) [inline src/GallerySwiper.jsx]: @sylvesteraswin any idea how to create id names so there are no problems with multiple instances?

Comment (lucidNTR) [reply src/GallerySwiper.jsx]: if you dont have a better idea i will generate https://lodash.com/docs/4.17.2#uniqueId a unique id on the "component will mount" event and add that to the  base_class

Comment (sylvesteraswin) [reply src/GallerySwiper.jsx]: @lucidNTR you just use a simple uid using a random number and add to to state when the component mounts.

**Review by sylvesteraswin — CHANGES_REQUESTED** [review]

File: src/GallerySwiper.jsx [hunk 2]
```diff
@@ -940,7 +941,7 @@ class GallerySwiper extends Component {
         if (!disableArrowKeys) {
             events = (
                 <AttachHandler
-                    target={BASE_CLASS}
+                    target={BASE_CLASS + this.state.id}
```
Comment (sylvesteraswin) [inline src/GallerySwiper.jsx]: please use ` to concat variables.

**Review by sylvesteraswin — CHANGES_REQUESTED** [review]

**Comment by lucidNTR** [comment]
@sylvesteraswin used math random for id generation as you requested and string literals, ready for review now

**Comment by lucidNTR** [comment]
also sorry for the typos in the commit message, just saw them :-(

**Review by sylvesteraswin — APPROVED** [review]

**Pull request merged**